src/main/resources/META-INF/rewrite/jakarta-faces-4.yml

```diff
@@ -0,0 +1,450 @@
+name: org.openrewrite.java.migrate.jakarta.JakartaFacesXhtml
+displayName: Faces XHTML migration for Jakarta EE 10
+description: Find and replace legacy JSF namespaces and javax references with Jakarta Faces values in XHTML files.
+tags:
+  - jakarta
+  - faces
+  - jsf
+recipeList:
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsf/html"
+      replace: "jakarta.faces.html"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsf/html"
+      replace: "jakarta.faces.html"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsf/facelets"
+      replace: "jakarta.faces.facelets"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsf/facelets"
+      replace: "jakarta.faces.facelets"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsf/core"
+      replace: "jakarta.faces.core"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsf/core"
+      replace: "jakarta.faces.core"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsp/jstl/core"
+      replace: "jakarta.tags.core"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsp/jstl/core"
+      replace: "jakarta.tags.core"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsf/composite"
+      replace: "jakarta.faces.composite"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsf/composite"
+      replace: "jakarta.faces.composite"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsf/passthrough"
+      replace: "jakarta.faces.passthrough"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsf/passthrough"
+      replace: "jakarta.faces.passthrough"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsp/jstl/functions"
+      replace: "jakarta.tags.functions"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsp/jstl/functions"
+      replace: "jakarta.tags.functions"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://java.sun.com/jsf"
+      replace: "jakarta.faces"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: "http://xmlns.jcp.org/jsf"
+      replace: "jakarta.faces"
+      filePattern: '**/*.xhtml'
+  - org.openrewrite.text.FindAndReplace:
+      find: javax.
+      replace: jakarta.
+      filePattern: '**/*.xhtml'
```

Comment:
Are we sure this is both necessary and safe at this point after the earlier replacements? I'm asking since it's potentially quite a wide match, which could lead to unexpected results if applied to broadly.